amundsen_application/static/js/ducks/tableMetadata/types.ts

```diff
@@ -154,3 +155,43 @@ export interface UpdateTableOwnerResponse {
     owners: OwnerDict;
   };
 }
+
+export enum GetTableLineage {
+  REQUEST = 'amundsen/tableMetadata/GET_TABLE_LINEAGE_REQUEST',
+  SUCCESS = 'amundsen/tableMetadata/GET_TABLE_LINEAGE_SUCCESS',
+  FAILURE = 'amundsen/tableMetadata/GET_TABLE_LINEAGE_FAILURE',
+}
+export interface GetTableLineageRequest {
+  type: GetTableLineage.REQUEST;
+  payload: {
+    key: string;
+  };
+}
+export interface GetTableLineageResponse {
+  type: GetTableLineage.SUCCESS | GetTableLineage.FAILURE;
+  payload: {
+    lineage: Lineage;
+    status: number;
+  };
+}
+
+export enum GetColumnLineage {
+  REQUEST = 'amundsen/tableMetadata/GET_COLUMN_LINEAGE_REQUEST',
+  SUCCESS = 'amundsen/tableMetadata/GET_COLUMN_LINEAGE_SUCCESS',
+  FAILURE = 'amundsen/tableMetadata/GET_COLUMN_LINEAGE_FAILURE',
+}
+export interface GetColumnLineageRequest {
+  type: GetColumnLineage.REQUEST;
+  payload: {
+    key: string;
+    columnName: string;
+  };
```

Comment:
Mentioned below with the reason.
We probably want to track this action like in https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/bookmark/reducer.ts#L28-L36